Bestpractices.md

```diff
@@ -26,9 +56,9 @@ If you aren't sure about any of the points below, if you have questions about th
 
 * Do not alter the behavior of Javascript classes that will be used outside of your app such as `String` and `Array`.
 
-## Zendesk Apps: Best Practices
+### Best Practices
 
-* Make sure you define an App version. An App version will help you, and anyone else using the App keep track of what is installed. This can be particularly helpful if a bug is found and you need to provide a newer version of an App.
+* Make sure you define an app version. An app version will help you, and anyone else using the app keep track of what is installed. This can be particularly helpful if a bug is found and you need to provide a newer version of an app.
 
 * Define CSS in `app.css`, instead of leaving them in the templates.
```

Comment:
I thought this was a blocker, as in we won't approve any app with css or js in the template.